pkg/reconciler/resources.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"time"
+
+	"knative.dev/pkg/kmeta"
+)
+
+const (
+	// MinimumResourceAge is the age at which resources stop being IsYoungResource.
+	MinimumResourceAge = 30 * time.Second
```

Comment:
Does this need to be exported?

Reply:
technically no, but I thought it might be useful if folks needed to use it for unit testing.T

Reply:
Wow, the `T` was a typo, but it just worked out so well I'm not even going to fix it 😂

Reply:
If it seems like it will be useful in the future we can export it then, in general I don't like to export things unless we know we want to make it part of the public Go API we expose to users.

Reply:
done